docs/modules/ROOT/pages/general-info/supported-platforms.adoc

```diff
@@ -6,20 +6,24 @@ details the JVMs currently supported by the Payara Platform Community (both Paya
 
 == Supported Java Virtual Machines
 
-The Payara Platform currently supports the following Java Virtual Machines:
+The Payara Platform currently runs the following Java Virtual Machines:
 
-* Oracle JDK8 (u162+), Oracle JDK 11 (11.0.5+)
-* Azul Zulu JDK8 (u162+), Azul Zulu JDK 11 (11.0.5u10+)
-* OpenJDK JDK8 (u162+), OpenJDK 11 (11.0.5+)
-* All other JVMs based on OpenJDK 8u162+ or 11.0.5+ (for example Corretto JDK, Liberica JDK, AdoptOpenJDK)
+* Azul Zulu JDK: 8 (u162+), 11 (11.0.5u10+)
+* Oracle JDK: 8 (u162+), 11 (11.0.5+)
+* Amazon Corretto: 8, 11 (11.0.5+)
+* Adopt Open JDK: 8, 11 (11.0.5+)
+* Adopt Open JDK with Eclipse Open J9
+* Any other JVM based on OpenJDK 8u162+ or 11.0.5+
 
-TLS 1.3 is supported on JDK 8 with Azul Zulu 1.8.222+, Open JDK 8u262+ based builds, and all JDK 11 versions.
+TLS 1.3 is supported on JDK 8 with Azul Zulu 1.8.222+, Open JDK 8u262+ based builds, and all version 11 of all above JDKs.
 
-NOTE: A separate Payara Server Blue distribution is no longer required for IBM Java SDK.
+== Supported Architectures
+
+The Payara Platform currently runs on the `x64` and `arm64` CPU architectures. 
```

Comment:
The JDK for ARM is a specific download, so shouldn't it be mentioned in the Supported JVMs above?

Reply:
I think it would be too complicated. We would need to list that we support ARM versions of all the supported JDKs. Do you know which distributions provide an ARM version? I think it's easier to say that ARM versions of all the supported JDKs are supported.

Reply:
I find it confusing we say here ARM is supported but there is no ARM JDK version mentioned as supported JDK.

You have versions from Oracle, Azul, and Bellsoft and AWS EC2 A1 instances have also an ARM.

And probably not all ARM versions are supported like very old ones like ARM 8u6, or ARM 13.

Reply:
There are no ARM-specific versions of the JDKs. For a specific version, there are x64 and arm variants to download. So it should be enough to mention that Payara supports x64 and arm64 variants. We can move this info right under the list of supported JVMs though, I'll do that.